raw_data_QA/check_raw_case_death_data.py

```diff
@@ -371,11 +366,6 @@ def out_path(foldername, args):
     return str(args.output_dir + "/" + args.output_data_dir + "/" + foldername)
```

Comment:
I was expecting this to be
`return str(args.output_data_dir + "/" + foldername)`
Is there a reason to keep the output_dir prefix?

Reply:
changed